app/services/search/client.rb

```diff
@@ -0,0 +1,53 @@
+module Search
```

Comment:
Do we need to require elasticsearch here like we did in the initializer?
```
require "elasticsearch"
```

Reply:
Not really, as it's autoloaded as most gems. Just ried `rails runner "puts Elasticsearch::Client"` to make sure